### PR TITLE
Add a custom flag to disable marking of copy assemblies

### PR DESCRIPTION
**Pull request by vitek-karas**
For https://github.com/dotnet/linker/issues/2089

Enabled by `--custom-data DisableMarkingOfCopyAssemblies=true` on the command line. Assumes that ALL assemblies on the input are in "copy" action (it doesn't validate this fact).

It disables marking basically fully - linker will go over all assemblies, and process them in "copy" mode (copy the original file over) and will call all the custom steps and so on, but it will do no marking (or very little, depends on descriptors and such which this doesn't disable).

This is intentionally non-discoverable feature, to be used only by the mono AOT toolchain.

/cc @rolfbjarne

### CODE REVIEW

**Comment by vitek-karas** [comment]
@agocke for triage - this should go into 6.0.2xx and maybe potentially even sooner depending on mono needs.

File: src/linker/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (sbomer) [inline src/linker/Properties/launchSettings.json:1]: Left-over file I think

**Review by sbomer — APPROVED** [review]
LGTM

**Review by rolfbjarne — APPROVED** [review]

**Comment by vitek-karas** [comment]
@agocke - can you please confirm that it's OK to merge this to 6.0.2xx? (since it's kind of the first real linker fix for servicing).

**Comment by agocke** [comment]
LGTM

**Pull request merged**